server/index.js

```diff
@@ -1,9 +1,53 @@
-const express = require('express')
-const app = express()
-const port = 3000
+const express = require('express');
+const app = express();
+const port = 3000;
+const db = require('../database/index.js');
+const parser = require('./parser.js');
 
 app.use(express.static(__dirname + '/../client/dist'));
 
+app.get('/community/:projectId', (req, res) => {
+  let projectId = req.params.projectId, cityIds = [], countryChance = [], cityChance = [], countryNames = [], cityNames = [], finalLocations;
+
+  Promise.resolve(db.getLocations(projectId))
```

Comment:
Promise.resolve may be redundant, since you're already returning a promise from db.getLocations()